docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -27,11 +27,11 @@ Once your new application is saved a client ID is generated, add the client ID a
 
 ## Build
 
-From a terminal at your application's root directory, `npm run build`. This will create a deployment-ready build in the "build" folder within the application's root directory. It is not necessary to build the application during development.
+From a terminal at your application's root directory, `npm run build`. This will create both the frontend in the "build" folder and backend in the "lib" folder within the application's root directory. It is not necessary to build the application during development.

```

Comment:
```suggestion
From a terminal at your application's root directory, `npm run build`. This will create both the frontend in the "build" folder and backend in the "lib" folder within the application's root directory. It is not necessary to build the frontend during development.
```

Do need to have the backend piece built for development.

Reply:
Good point, applied the suggestion.